sentry/src/test/resources/envelope-transaction-with-sample-rate.txt

```diff
@@ -0,0 +1,3 @@
+{"event_id":"3367f5196c494acaae85bbbd535379ac","trace":{"trace_id":"b156a475de54423d9c1571df97ec7eb6","public_key":"key","sample_rate":"0.00000021"}}
```

Comment:
Can we add all the other missing fields and assert them as well? such as `release`, `environment`, user, etc...

Reply:
Updated